roles/prometheus/meta/argument_specs.yml

```diff
@@ -112,6 +112,7 @@ argument_specs:
       prometheus_scrape_configs:
         description:
           - "Prometheus scrape jobs provided in same format as in the L(official docs,https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config)"
+          - "This feature is available starting from Prometheus v2.43.0."
```

Comment:
```suggestion
```

Reply:
Snap, that one’s particularly embarrassing. Sorry and thank you for pointing it out!